roamance/src/components/profile/profile-header.tsx

```
@@ -0,0 +1,232 @@
+                    src={user?.profileImage || undefined}
+                    alt={user?.name || 'User'}
+                    className="object-cover"
+                  />
+                  <AvatarFallback className="bg-gradient-to-br from-ocean to-ocean-dark text-2xl md:text-3xl text-white font-light">
+                    {getInitials(user?.name)}
+                  </AvatarFallback>
+                </Avatar>
+                <div
+                  className={`absolute inset-0 bg-black/30 rounded-full transition-all duration-300 flex items-center justify-center backdrop-blur-sm ${isUploading ? 'opacity-100' : 'opacity-0 group-hover:opacity-100'} ${isUploading ? 'cursor-wait' : 'cursor-pointer'}`}
+                  onClick={() => !isUploading && setIsUploadDialogOpen(true)}
+                >
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)